pino_milestone_1.urdf

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<robot name = "pino_first_try">
+
+  <material name="violet">
+    <color rgba="0.5 0 1 1"/>
+  </material>
+
+  <material name="pink">
+    <color rgba="1 0.75 0.796 1"/>
+  </material>
+  
+  <link name="base_link">
+    <visual>
+      <geometry>
+          <box size="0.1 0.1 0.1"/>
+      </geometry>
+      
+    </visual>
+  </link>
+
+  <link name="arm_1">
+    <visual>
+      <geometry>
+        <box size="1.8 0.02 0.1"/>
+      </geometry> 
+      <origin xyz="0.9 0.0 0.0"/>
+      <material name="violet"/>
+    </visual>   
+  </link>
+
+  <link name="arm_2">
+    <visual>
+      <geometry>
+          <box size="1 0.02 0.1"/>
+      </geometry>  
+      <origin xyz="0.5 0.0 0.0" rpy="0.0 0.0 0"/>
+      <material name="pink"/>  
+    </visual>
+  </link>
+
+  <joint name="base_to_arm_1" type="revolute">
+    <axis xyz="0 0 1"/>
```

Comment:
Your rotation is not in the correct axis. This applies to the other joint as well. Take a look at the urdf in urdf-viz again, and adjust the axes of rotation accordingly (I'm deliberately not giving the answer, but feel free to ask if you're not sure). Keep in mind that the axis colours are mapped such that RGB -> XYZ.